### ISSUE TITLE
LastPass form fills for credit cards don't work

### ISSUE THREAD

**Issue by kalifg** [brave/browser-laptop#5968]
**Did you search for similar issues before submitting this one?**
Yes

**Describe the issue you encountered:**
When I use the LastPass form fill functionality, sometimes name is filled but most of the time not.  Address, phone, credit card, etc don't get filled

**Expected behavior:**
I expect my credit card info to be filled into the form

- Platform (Win7, 8, 10? macOS? Linux distro?):
macOS 10.12.1

- Brave Version:
0.12.10

- Steps to reproduce:
    1. Have LastPass enabled, with at least one form fill available with credit card info
    2. Find a shopping cart form to enter credit card info in. 
    3. Try to fill form by either using the icon on the right of the field or right-clicking and choosing `LastPass -> Fill Forms -> <form data name> -> Fill Form`

- Screenshot if needed:
![screen recording 2016-12-01 at 10 32 am](https://cloud.githubusercontent.com/assets/487194/20802580/1407d030-b7b2-11e6-995e-b0b4584f2244.gif)

- Any related issues:

**Comment by SimonWaters** [comment]
Have you reproduced on another browser (say Chrome) to exclude a LastPass issue?

**Comment by bsclifton** [comment]
@SimonWaters great point :smile: 

@kalifg can you please try with another browser to see if it's an issue with LastPass? You can view the version of LastPass by going to about:extensions (mine is showing 4.1.28)

**Comment by bsclifton** [comment]
yikes! [Looking at the Chrome store](https://chrome.google.com/webstore/detail/lastpass-free-password-ma/hdokiejnpimakedhajhdlcegeplioahd?hl=en-US), our version of LastPass is slightly behind (latest is 4.1.33)

cc: @bbondy

**Comment by jolson222** [comment]
I have the same issue.  I verified that the form fill works in the Pale Moon browser, but I can't get it to work in Brave.  LastPass version shows as 4.1.28.

**Comment by jolson222** [comment]
Okay, I also checked form fill functionality in Chrome.  It worked there too.

**Comment by kalifg** [comment]
Sorry for the delay, been away from a computer for a while.

The site does work in Safari, although it's worth mentioning that a dialog popped up asking me if I wanted to fill in the information first.  That dialog does not appear in Brave.

I tried a few other sites and I did find some where in Chrome or Safari the fill would not function correctly the first time, but if I told it to again it would work.  In Brave nothing would happen after repeated attempts.
